src/index.html

```diff
@@ -66,6 +67,9 @@
         logo-link="https://www.esri.com/en-us/arcgis/products/arcgis-instant-apps/overview"
         info-button="true"
         info-is-open="true"
+
+        custom-header-html="<div>My custom Header test <img onload=alert('hacked'); src='https://picsum.photos/50'/> </div>"
```

Comment:
Would it be possible to remove the markup from the sample app? Don't think it's necessary to include this in the sample for the component library